src/ZPublisher/WSGIPublisher.py

```diff
@@ -201,7 +202,9 @@ def _publish_response(request, response, module_info, _publish=publish):
             response.setBody(view())
             return response
 
-        raise exc
+        # Reraise the exception, preserving the original traceback
+        t, v, tb = sys.exc_info()
```

Comment:
Hmm, maybe it is just old scar tissue, but I still get the heebie-jeebies seeing the traceback bound to a local variable:  I would instinctively have written that as:

        reraise(exc.__class__, exc, sys.exc_info()[-1])